src/components/Tooltip/Arrow.tsx

```diff
@@ -11,38 +10,38 @@ type ArrowsProps = {
 
 const Base = styled.div<ArrowsProps>`
   position: absolute;
-  ${({ width }) => width && `width: ${width}px`};
-  ${({ width }) => width && `height: ${width}px`};
-  background: ${({ background }) => background || colors.blue};
+  ${({ width }) => width && `width: ${width}px;`};
+  ${({ width }) => width && `height: ${width}px;`};
+  background: ${(props) => props.background || props.theme.palette.secondary.main};
```

Comment:
You can destructure props here like this
```suggestion
  background: ${({ background, theme }) => background || theme.palette.secondary.main};
```